apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -519,6 +605,39 @@ const EventTypePage = (props: EventTypeSetupProps) => {
           }}
         />
       ) : null}
+
+      <Dialog open={unsavedDialogOpen} onOpenChange={setUnsavedDialogOpen}>
+        <ConfirmationDialogContent
+          isLoading={updateMutation.isLoading}
+          variety="danger"
+          cancelBtnText="Discard Changes"
+          // confirmBtnText="Keep Editing"
+          title={t(`delete${"_managed"}_event_type`)}
+          confirmBtnText={t(`Save Changes`)}
+          loadingText={t(`Save Changes`)}
+          onCancel={(e) => {
+            setShouldWarnUnsaved(false);
+            e.preventDefault();
+            router.push(newRoute);
+          }}
+          onConfirm={async (e) => {
+            setShouldWarnUnsaved(false);
+            e.preventDefault();
+            await handleSubmit(formMethods.getValues());
+            router.push(newRoute);
+          }}>
+          <p className="mt-5">
+            <Trans i18nKey="SaveChanges" components={{ li: <li />, ul: <ul className="ml-4 list-disc" /> }}>
+              <ul>
+                <li>Members assigned to this event type will also have their event types deleted.</li>
+                <li>
+                  Anyone who they&apos;ve shared their link with will no longer be able to book using it.
+                </li>
+              </ul>
+            </Trans>
```

Comment:
```suggestion
            <Trans i18nKey="SaveChanges" components={{ li: <li />, ul: <ul className="ml-4 list-disc" /> }}>
              <ul>
                <li>Members assigned to this event type will also have their event types deleted.</li>
                <li>
                  Anyone who they&apos;ve shared their link with will no longer be able to book using it.
                </li>
              </ul>
            </Trans>
```
I think this text is wrong

Reply:
fixed.